README.md

```diff
@@ -49,70 +68,135 @@ const client = new Realtime(ABLY_API_KEY);
 const spaces = new Spaces(client);
 ```
 
-Create a new space instance and have members enter that space:
-
-```ts
-const space = spaces.get('mySpace');
-space.enter({
-  username: 'Example User',
-  avatar: 'https://example.com/users/1234.png',
-});
-```
+### Space membership
 
-Subscribe to updates from other members in the space. The `membersUpdate` event will fire when a member's connection status updates, e.g. when a member joins or leaves the space:
+A space is the virtual area of an application you want to monitor, such as a web page, or slideshow. Create a space and subscribe to events for when clients enter and leave it. Space membership is used to build avatar stacks and display which members are online within a space.
```

Comment:
```suggestion
A space is the virtual collaborative area of an application you want to monitor. You can decide what constitutes a space in your app, it could be the entire app or just a specific artefact such as a web page, or slidedeck. Create a space and subscribe to events for when clients enter and leave it. Space membership is used to build avatar stacks and display which members are online within a space.
```